src/module/actor/sheet/base.ts

```diff
@@ -123,6 +123,7 @@ export abstract class ActorSheetPF2e<TActor extends ActorPF2e> extends ActorShee
             totalWealth,
             totalWealthGold,
             inventory: this.prepareInventory(),
+            hasInventory: this.actor.inventory.size === 0 ? false : true,
```

Comment:
```suggestion
            hasInventory: !!this.actor.inventory.size,
```

Reply:
TBH you don't even need this, in the sheet if you do `{{#if document.inventory.size}}` that's the same result.

Reply:
Updated to use suggested simplification. Thanks.